### PR TITLE
Remove outdated connection between name and machine name

### LINKED ISSUE #3481: Notice: Undefined index: name in InputHandler.php
When performing `drush generate test-unit` the following notice occurs:

Undefined index: name vendor/drush/drush/src/Commands/generate/Helper/InputHandler.php on line 102

```
            $default_machine_name = function ($vars) use ($modules, $root_directory) {
                $machine_name = array_search($vars['name'], $modules);
                return $machine_name ?: Utils::human2machine(isset($vars['name']) ? $vars['name'] : $root_directory);
            };
```

### PR DESCRIPTION
**Pull request by Chi-teck**
This should resolve #3481.

### CODE REVIEW

File: src/Commands/generate/Helper/InputHandler.php [hunk 1]
```diff
@@ -89,17 +89,16 @@ protected function preprocessQuestions(array &$questions, $destination)
             // Name question.
             if (isset($questions['name'])) {
                 $questions['name']->setAutocompleterValues($modules);
-                $default_name = function ($vars) use ($root_directory) {
-                    return Utils::machine2human(isset($vars['machine_name']) ? $vars['machine_name'] : $root_directory);
+                $default_name = function () use ($root_directory) {
+                  return $root_directory ? Utils::human2machine($root_directory) : '';
```
Comment (weitzman) [inline src/Commands/generate/Helper/InputHandler.php]: Is the anonymous function needed? How about
```
$default_name = $root_directory ? Utils::human2machine($root_directory) : '';
```

Comment (Chi-teck) [reply src/Commands/generate/Helper/InputHandler.php]: Surely, it is not needed now.

File: src/Commands/generate/Helper/InputHandler.php [hunk 2]
```diff
@@ -89,18 +89,15 @@ protected function preprocessQuestions(array &$questions, $destination)
             // Name question.
             if (isset($questions['name'])) {
                 $questions['name']->setAutocompleterValues($modules);
-                $default_name = function ($vars) use ($root_directory) {
-                    return Utils::machine2human(isset($vars['machine_name']) ? $vars['machine_name'] : $root_directory);
+                $default_name = function () use ($root_directory) {
```
Comment (weitzman) [inline src/Commands/generate/Helper/InputHandler.php]: Looks like this anon function can be removed as well.

Comment (Chi-teck) [reply src/Commands/generate/Helper/InputHandler.php]: Done.

**Pull request merged**